Model/Config/Source/Payment/Settings.php

```diff
@@ -85,6 +85,8 @@ public function getNotificationEndpoint()
         try {
             return $this->_storeManager->getStore()->getBaseUrl() . 'snap/payment/notification';
         } catch (NoSuchEntityException $e) {
+            $this->midtransLogger->midtransError('GetNotificationEndpointFunction: ' . $e);
```

Comment:
I think we dont need to log it, because this block can get called for every time Config page is opened (or maybe everytime transaction is created) right? it may pollute the Log file.

Reply:
Nope pak, it will record to Midtrans error.log when function  `$this->_storeManager->getStore()->getBaseUrl()` facing some error